internal/rootcoord/import_manager.go

```diff
@@ -112,9 +112,17 @@ func newImportManager(ctx context.Context, client kv.MetaKv,
 func (m *importManager) init(ctx context.Context) {
 	m.startOnce.Do(func() {
 		// Read tasks from Etcd and save them as pending tasks or working tasks.
-		m.loadFromTaskStore()
+		err := m.loadFromTaskStore()
+		if err != nil {
+			log.Error("importManager init failed, read tasks from Etcd failed, about to panic")
+			panic(err)
+		}
 		// Send out tasks to dataCoord.
-		m.sendOutTasks(ctx)
+		err = m.sendOutTasks(ctx)
+		if err != nil {
+			log.Error("importManager init failed, send out tasks to dataCoord failed, about to panic")
+			panic(err)
```

Comment:
sendOutTasks no need to panic.

Reply:
Agreed. Just logging an error should be good enough here